user_visit/models.py

```diff
@@ -42,6 +42,9 @@ def build(self, request: HttpRequest, timestamp: datetime.datetime) -> UserVisit
             context=REQUEST_CONTEXT_EXTRACTOR(request),
         )
         uv.hash = uv.md5().hexdigest()
+        uv.browser = uv.user_agent.get_browser()
+        uv.device = uv.user_agent.get_device()
+        uv.os = uv.user_agent.get_os()
```

Comment:
I appreciate none of these should overflow the max_length, but it would be good to ensure it doesn't happen:
```suggestion
        uv.browser = uv.user_agent.get_browser()[:200]
        uv.device = uv.user_agent.get_device()[:200]
        uv.os = uv.user_agent.get_os()[:200]
```